src/install/vm/whatsNext.mdx

```diff
@@ -0,0 +1,12 @@
+---
+componentType: default
+headingText: What's next?
+---
+
+#### See your app data in the UI.
```

Comment:
wow I had no idea we use level 4 headers! These should probably be level 2 headers